app/errors.js

```diff
@@ -29,3 +29,16 @@ exports.albumsApiRedirection = message => internalError(message, exports.ALBUMS_
 
 exports.ALBUM_ORDER_ERROR = 'album_order_error';
 exports.albumOrderError = message => internalError(message, exports.ALBUM_ORDER_ERROR);
+
+exports.errorName = {
+  albumNotFound: 'ALBUM_NOT_FOUND'
+};
+
+const errorType = {
+  ALBUM_NOT_FOUND: {
+    message: 'Album not found',
+    statusCode: 404
+  }
+};
+
+exports.getErrorCode = errorName => errorType[errorName];
```

Comment:
Export directly error as `exports.ALBUM_NOT_FOUND = { message: 'Album not found', statusCode: 404 }`, for avoid this functions and unnecessary boilerplate code